### PR TITLE
Issue #1818: Add new function for getting list of common HTML elements.

### LINKED ISSUE #1818: [DX] Allow consistent HTML element selection in UIs
Both Views and Blocks allow you to rewrite their markup, but they allow different (and sometimes inadequate) tags. Let's make these consistent and complete the list of allowed options.

In views, the list of allowed elements is stored in config `views.settings.json`.  

```
'views_field_rewrite_elements' => array(
      '' => t('- Use default -'),
      '0' => t('- None -'),
      'div' => 'DIV',
      'span' => 'SPAN',
      'h1' => 'H1',
      'h2' => 'H2',
      'h3' => 'H3',
      'h4' => 'H4',
      'h5' => 'H5',
      'h6' => 'H6',
      'p' => 'P',
      'strong' => 'STRONG',
      'em' => 'EM',
    ),
```

We could move this variable out of views` and use it in all locations: views, layouts, and blocks.

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/1818

### CODE REVIEW

**Review by klonos — APPROVED** [review]
Code looks good 👍

**Comment by docwilmot** [comment]
Would an `alter()` be warranted here, to allow other modules to add HTML elements?

**Comment by ghost** [comment]
Good idea @docwilmot! PR updated (please let me know if I did it wrong).

File: core/modules/layout/layout.admin.inc [hunk 1]
```diff
@@ -2974,17 +2974,10 @@ function layout_configure_region_page($form, &$form_state, Layout $layout, $rend
...
-    'div' => 'DIV',
-    'nav' => 'NAV',
-    'aside' => 'ASIDE',
-    'section' => 'SECTION',
-  );
   $form['element'] = array(
     '#title' => t('HTML element for inner wrapper'),
     '#type' => 'select',
-    '#options' => $options,
+    '#options' => array('' => 'No inner wrapper') + backdrop_html_elements(array('layout')),
```
Comment (klonos) [inline core/modules/layout/layout.admin.inc]:
```suggestion
    '#options' => array('' => t('No inner wrapper')) + backdrop_html_elements(array('layout')),
```

**Review by klonos — APPROVED** [review]
LGTM 👍

**Pull request closed without merging**